config/example_argos.json

```diff
@@ -0,0 +1,23 @@
+{
+	"type": "indexed",
+	"dimensionality": 1,
+	"chunks": [],
+	"dataset": {
+		"dimension_map": {
+			"time": "t_time"
+		},
+		"variable_map_comment":"mapping from external variable name to standard variable names see https://pcmdi.llnl.gov/mips/cmip3/variableList.html",
+		"variable_map": {
+			"DATIM": "time",
+			"LAT": "latitude",
+			"LON": "longitude",
+			"SST": "sea_surface_temperature",
+			"PPRES": "pressure"
+		},
+		"coord_vars": ["time","latitude","longitude"],
+		"keep_all_vars": "False"
```

Comment:
This new key will have to be added to the [COAsT-Site documentation](https://github.com/British-Oceanographic-Data-Centre/COAsT-site) . In [configuration_files.md](https://github.com/British-Oceanographic-Data-Centre/COAsT-site/blob/develop/content/en/docs/Examples/configuration_files.md)

Reply:
@roje-bodc hi, can you do this ? I don't know how to update COAsT-Site documentation and may not have the permissions. Thanks.